pytorch_lightning/callbacks/pt_callbacks.py

```diff
@@ -177,6 +178,17 @@ def __init__(self, filepath, monitor='val_loss', verbose=0,
                  save_best_only=True, save_weights_only=False,
                  mode='auto', period=1, prefix=''):
         super(ModelCheckpoint, self).__init__()
+        if (
+            save_best_only and
+            os.path.exists(filepath) and
```

Comment:
use just `os.path.isdir(filepath)` instead of `os.path.exists(filepath) and os.path.isdir(filepath)`

Reply:
Fixed!